src/editorial-source-components/SvgCrossRound.tsx

```diff
@@ -0,0 +1,20 @@
+// This has been taken from the Guardian Source design system:
+// https://github.com/guardian/csnx/blob/63a2a71585cf4f98b1d84e3a40aa1e8fa19f70c4/libs/%40guardian/source-react-components/vendor/icons/SvgCrossRound.tsx
+// Not available on our current version of Source, but can be pulled in when we upgrade.
```

Comment:
It sounds like there's an active desire not to use Source because we don't want that coupling, so maybe we don't need this comment?

Reply:
I've removed the comment as you both seem to agree (https://github.com/guardian/prosemirror-elements/pull/286/commits/35bac7740caea55e01cdea3500ed64c48fde20b6), however my personal thoughts are if we take code from a different project we should reference the source (not the Source, in this case, just where the code was lifted from)